<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemSettings2</class>
 <widget class="QDialog" name="DialogSystemSettings2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget {
background-color: rgb(99,99,99); */
/*background-color: rgb(169,169,29); */
/*background-color: rgb(29,200,200); */
/*color: rgb(229,229,229); 
}*/

QPushButton {
background-color: rgb(129,129,129); 
color: rgb(229,229,229); 
border: 1px solid black;
border-radius: 4px;
}

/*QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgba(60, 186, 162, 255), stop:1 rgba(98, 211, 162, 255))
}*/

/*QPushButton {
     background-color: #3cbaa2; 
     border: 1px solid black;
     border-radius: 4px;
}*/

/*QPushButton:disabled {
    background-color: rgb(170, 170, 127)
}*/

QFrame {
background-color: rgb(69,69,69); 
color: rgb(229,229,229); 
}

QLabel {
background-color: rgb(119,119,119); 
color: rgb(229,229,229); 
}

/*QTreeWidget {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(96,96,96), stop: 1 rgb(196,196,196));
color:rgb(232,232,232); 
border: 2px solid rgb(190,220,15);
border-radius: 8px;border-style:solid; 
}*/

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>System Settings</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonSystem">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>System</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConfig">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>I/O Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUserInterface">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>User Interface</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonPowerOn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Power On</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRestoreDefault">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Restore Defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAlignments">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Alignments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonLicensing">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Licensing...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSecurity">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Security</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDiagnostics">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Diagnostics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonService">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Service</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="40,1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>System</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonHelp">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExit">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedSystem">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageSystem">
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,3" columnstretch="2,1" rowminimumheight="1,1" columnminimumwidth="1,1">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Windows Control</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Application Controls</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Sound</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageConfig">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Config</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageUserInterface">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>UserInterface</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pagePowerOn">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>PowerOn</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageRestoreDefaults">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Restore Defaults</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageAlignments">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Alignments</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageLicensing">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Licensing</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSecurity">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Security</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageDiagnostics">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Diagnostics</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageService">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Service</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
